styles.css

```diff
@@ -0,0 +1,95 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.body {
+  font-family: "Montserrat", sans-serif;
+  font-size: 16px;
+}
+
+.home-page {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.h1, .h2, .h3 {
```

Comment:
There are no classes of this name in your html: use the type selectors h1 etc (no dot)